caffe2/core/tensor.cc

```diff
@@ -16,8 +16,6 @@ CAFFE2_DEFINE_int64(
 
 namespace caffe2 {
 
-CAFFE_DEFINE_KNOWN_TYPE(Tensor);
```

Comment:
Err... you can't delete this until Tensor unifies, right?

Reply:
I combined the `CAFFE_DECLARE_KNOWN_TYPE` and `CAFFE_DEFINE_KNOWN_TYPE` macros into a `CAFFE_PREALLOCATED_KNOWN_TYPE`. That one still exists for `Tensor`, so everything is fine.